src/scr.c

```diff
@@ -678,18 +678,32 @@ static int scr_get_params()
   /* if not enabled, bail with an error */
   if (! scr_enabled) {
     scr_param_finalize();
+    if (scr_my_rank_world == 0) {
+      scr_dbg(1, "SCR_ENABLE=%d", scr_enabled);
+    }
     return SCR_FAILURE;
   }
+  else {
+    if (scr_my_rank_world == 0) {
+      scr_dbg(1, "SCR_ENABLE=%d", scr_enabled);
+    }
+  }
 
   /* set debug verbosity level */
   if ((value = scr_param_get("SCR_DEBUG")) != NULL) {
     scr_debug = atoi(value);
   }
+  if (scr_my_rank_world == 0) {
+     scr_dbg(1, "SCR_DEBUG=%d",scr_debug);
+  }
```

Comment:
Just for a style thing, we typically use 2 spaces for indents in the code, but it looks like you might have 3 for all of the scr_dbg lines.  Would you please update all of these lines to use 2 spaces instead?
```suggestion
  if (scr_my_rank_world == 0) {
    scr_dbg(1, "SCR_DEBUG=%d", scr_debug);
  }
```